@here/harp-examples/src/datasource_object-picking.ts

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
@pit-rpg  added the same example in https://github.com/heremaps/harp.gl/pull/734.
So is @pit-rpg PR obsolete?

Reply:
@ninok No, i've based this on #734 only to be able to showcase the improvement.
Will remove this commit from PR and wait for #734.